backend/src/BuiltinExecution/Libs/UInt8.fs

```diff
@@ -0,0 +1,382 @@
+module BuiltinExecution.Libs.UInt8
+
+open FSharp.Control.Tasks
+open System.Threading.Tasks
+
+open System.Numerics
+
+open Prelude
+open LibExecution.RuntimeTypes
+open LibExecution.Builtin.Shortcuts
+
+module VT = ValueType
+module Dval = LibExecution.Dval
+
+let types : List<BuiltInType> = []
+let constants : List<BuiltInConstant> = []
+
+module ParseError =
+  type ParseError =
+    | BadFormat
+    | OutOfRange
+
+  let toDT (e : ParseError) : Dval =
+    let (caseName, fields) =
+      match e with
+      | BadFormat -> "BadFormat", []
+      | OutOfRange -> "OutOfRange", []
+
+    let typeName = TypeName.fqPackage "Darklang" [ "Stdlib"; "UInt8" ] "ParseError" 0
+    DEnum(typeName, typeName, [], caseName, fields)
+
+
+let fn = fn [ "UInt8" ]
+
+let fns : List<BuiltInFn> =
+  [ { name = fn "mod" 0
+      typeParams = []
+      parameters = [ Param.make "a" TUInt8 ""; Param.make "b" TUInt8 "" ]
+      returnType = TUInt8
+      description =
+        "Returns the result of wrapping <param a> around so that {{0 <= res < b}}.
+
+        The modulus <param b> must be greater than 0.
+
+        Use <fn UInt8.remainder> if you want the remainder after division, which has
+        a different behavior for negative numbers."
+      fn =
+        (function
+        | state, _, [ DUInt8 v; DUInt8 m ] ->
+          if m = 0uy then
+            Int.IntRuntimeError.Error.ZeroModulus
+            |> Int.IntRuntimeError.RTE.toRuntimeError
+            |> raiseRTE state.caller
+            |> Ply
+          else if m < 0uy then
```

Comment:
I hink this tests isn't necessary since  uint8 is always positive

Reply:
Yes, that's right